org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServers.java

```diff
@@ -257,7 +257,7 @@ public static class LanguageServerDocumentExecutor extends LanguageServers<Langu
 						LanguageServerPlugin.logError(e);
 					}
 				}
-				return CompletableFuture.completedFuture(wrapper);
+				return CompletableFuture.completedFuture(null);
```

Comment:
Hi @ahmedneilhussain , but t this point `wrapper` will be `null` because we checked on line 250, right?